docs/root/version_history/current.rst

```diff
@@ -11,7 +11,38 @@ Minor Behavior Changes
 ----------------------
 *Changes that may cause incompatibilities for some users, but should not for most*
 
+* access_log: log all header values in the grpc access log.
```

Comment:
Something went wrong here; I think you picked up a bunch of entries during a bad merge or something.